storage-model/src/main/java/no/unit/nva/publication/storage/model/daos/Dao.java

```diff
@@ -47,6 +47,11 @@ public final void setPrimaryKeyPartitionKey(String key) {
         // do nothing
     }
 
+    @Override
+    public final String getPrimaryKeySortKey() {
```

Comment:
Override and finalize all key methods so that we are sure that all subclasses have the same implementation.